.circleci/config.yml

```diff
@@ -724,13 +735,25 @@ jobs:
           name: Wait for SQS Queue to Empty
           command: |
             if [ "$MIGRATE_FLAG" == "true" ]; then
+              no_output_timeout: 5h
```

Comment:
I know I'm late to the party, but -- I'd like to learn more about this line, what it's doing, and why.  I don't object, just want to learn. :)

Reply:
since the migration can take upwards of an hour, and the current wait until complete script doesn't output anything, this increases the timeout time to a range we'd be able to feel confident.

i think the final version should probably bring this down to 2h _or_ have the script output status with how many messages it received. could help provide a status update.